nixos/modules/services/web-apps/hedgedoc.nix

```diff
@@ -3,33 +3,41 @@
 with lib;
 
 let
-  cfg = config.services.codimd;
+  cfg = config.services.hedgedoc;
+
+  name = if versionAtLeast config.system.stateVersion "21.03"
+    then "hedgedoc"
+    else "codimd";
```

Comment:
This totally does not at all depend on 21.03. I would just straight update everything to hedgedoc.

Reply:
Doing so, you would break existing installations, which already have a user named `codimd`, uploads, maybe even their sqlite db placed under `/var/lib/codimd`, which is owned by this pre-existing user.

Reply:
Should this appear in the changelog?